### PR TITLE
Book Suggestion Endpoint

### PR DESCRIPTION
**Pull request by cquiro**
## Summary

- Create BookSuggestionsController with a create method
- Pass 'optional: true' to the belongs_to relation in the book_suggestion.rb
- Create BookSuggestionSerializer
- Create a route to BookSuggestion#create (api/v1/book_suggestions) with a POST action
- Add new endpoint to the README

## Trello Card

https://trello.com/c/hUC7KqxF/29-crear-ruta-para-creaci%C3%B3n-de-un-booksuggestion

### CODE REVIEW

File: app/controllers/api/v1/book_suggestions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      private
+
+      def create_bs_params
+        params
+          .require(:book_suggestion)
+          .permit(:synopsis, :price, :author, :title, :link, :publisher, :year)
+      end
+
+      def new_book_suggestion
+        if current_user.present?
```
Comment (facucompe) [inline app/controllers/api/v1/book_suggestions_controller.rb:24]: Are you sure you can do this? I don't think you can call `current_user` if you are skiping `authenticate_user!`

**Review by facucompe — CHANGES_REQUESTED** [review]
Just check that

File: app/controllers/api/v1/book_suggestions_controller.rb (hunk 1 above)
Comment (cquiro) [reply app/controllers/api/v1/book_suggestions_controller.rb:24]: @facucompe yes, no problem. It still returns the logged in user or nil (when there is no logged in user).

**Review by facucompe — APPROVED** [review]
If the tests work, i guess it's ok

File: app/controllers/api/v1/book_suggestions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+module Api
+  module V1
+    class BookSuggestionsController < ApiController
+      skip_before_action :authenticate_user!, only: :create
+
+      def create
+        new_bs = new_book_suggestion
+
```
Comment (lmasello) [inline app/controllers/api/v1/book_suggestions_controller.rb]: Extra line

**Review by lmasello — APPROVED** [review]

**Pull request merged**